dist/api_data.py

```diff
@@ -476,6 +476,23 @@ def __ge__(self, other):
             above 0 configures periodic checkpoints''',
             min='0', max='100000'),
         ]),
+    Config('diagnostic', '', r'''
```

Comment:
I don't love the name {{diagnostic}} here - it overlaps with the HAVE_DIAGNOSTIC flag in an uncomfortable way. I'm not sure it's any better but maybe {{debug_mode}}?

Reply:
I can live with `debug_mode` or something similar. I am mildly concerned it may be confusing with the `debug_info` API. But maybe not. Perhaps it fits into a "family" of debugging related things even if they're config strings versus method names. I will make the naming change and see how it looks.